pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -62,3 +62,14 @@ def configure_ddp(self, model, device_ids):
             **self._ddp_kwargs,
         )
         return model
+
+    def on_before_forward(self, args: Any, model: LightningModule):
+        """
+        Override to handle custom input to device logic. For DDP, no logic is required as this is handled internally
+        within the DDP wrapper.
+        Args:
+            args: Inputs to the model.
+            model: Model to train.
+        Returns: args moved to correct device if needed.
```

Comment:
```suggestion
        
        Args:
            args: Inputs to the model.
            model: Model to train.
        
        Returns: args moved to correct device if needed.
```
can we add an example of such inputs?